src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/index.jsx

```diff
@@ -0,0 +1,154 @@
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import SchemaForm from 'platform/forms-system/src/js/components/SchemaForm';
+import FormButtons from '../../../components/FormButtons';
+import { LANGUAGES } from '../../../utils/constants';
+import * as actions from '../../redux/actions';
+import { getFormPageInfo } from '../../../utils/selectors';
+import { scrollAndFocus } from '../../../utils/scrollAndFocus';
+import ProviderSelectionField from './ProviderSelectionField';
+
+const initialSchema = {
+  type: 'object',
+  required: ['preferredLanguage', 'communityCareProvider'],
+  properties: {
+    communityCareSystemId: {
+      type: 'string',
+      enum: [],
+    },
+    preferredLanguage: {
+      type: 'string',
+      enum: LANGUAGES.map(l => l.id),
+      enumNames: LANGUAGES.map(l => l.text),
+    },
+    communityCareProvider: {
+      type: 'object',
+      properties: {
+        noProvider: {
+          type: 'boolean',
+        },
+        providerData: {
```

Comment:
This is basically the data structure for a provider from the facilities CC endpoint.